Makefile

```diff
@@ -0,0 +1,23 @@
+CONTAINER_NAME = dev
+
+define run_in_docker
+	docker build -t oi-bus-dev .
+	- docker run --rm --volume $(shell pwd):/app/oi-bus --workdir /app/oi-bus --name $(CONTAINER_NAME) -d -it oi-bus-dev
+	docker exec $(CONTAINER_NAME) $(1)
+	docker stop $(CONTAINER_NAME)
+endef
+
+
+build-docker:
+	$(call run_in_docker,make build)
+
+build:
+	dpkg-buildpackage -uc -us
+	mkdir -p out
+	mv ../*.deb out/
```

Comment:
this is wrong IMO - if you cloned the repo in a directory which already had a bunch of `.deb`s, this will copy those unrelated `.deb`s into  `oi-bus/out/`